kernels/tindex/TIndexKernel.cpp

```diff
@@ -194,6 +196,30 @@ StringList TIndexKernel::glob(std::string& path)
         filenames.push_back(filename);
     }
     globfree(&glob_result);
+#else
+    WIN32_FIND_DATA ffd;
+    HANDLE handle = FindFirstFile(path.c_str(), &ffd);
+
+    if (INVALID_HANDLE_VALUE == handle)
+        return filenames;
+
+    size_t found = path.find_last_of("/\\");
+    std::string dir = path.substr(0, found);
+
+    if (m_absPath)
+    {
+        TCHAR full_dir[4192] = TEXT("");
+        GetFullPathName(dir.c_str(), dir.length(), full_dir, NULL);
```

Comment:
This doesn't look right.  The second argument is supposed to be the size of the output buffer, which in this case is 4192.